File: Patterns/7. Pyramid Patterns/62. Hollow Pyramid/hp.java
import java.util.*;
public class hp{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            for(int j=1; j<=2*n-1; j++){
                if(i+j==6 || j-i==4 || i==n){
                    System.out.print("* ");
                }else{
                    System.out.print("  ");
                }
            }
            System.out.println();
        }
    }
}

/*
Output

        *         
      *   *       
    *       *     
  *           *   
* * * * * * * * * 

*/

File: Patterns/7. Pyramid Patterns/62. Hollow Pyramid/hp.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    for j in range(1,2*n):
        if(i+j==6 or j-i==4 or i==n):
            print("*",end=" ")
        else:
            print(end='  ')
    print()

'''
Output:

        *         
      *   *       
    *       *     
  *           *   
* * * * * * * * * 

'''